### ISSUE TITLE
Moefou4j

### ISSUE THREAD

**Issue by ThePreviousOne** [mariotaku/moe.fm#4]
could You please look into Updating this. I believe there are some Bugs in it, though I haven't confirmed yet, plus its 3 years out of date

**Comment by mariotaku** [comment]
I guess this could be rewritten using retrofit or something, old implementation is too old.

**Comment by mariotaku** [comment]
Do you interested in this moe.fm app?

**Comment by ThePreviousOne** [comment]
Yes. I'm trying to write a good, stable, version to eventually put on F-DROID

**Comment by mariotaku** [comment]
This project has not been finished yet, and service seems only provided in China, maybe can't be used by most people I guess.

**Comment by ThePreviousOne** [comment]
I am aware
